src/code_8040094_1.c

```diff
@@ -48,27 +48,27 @@ extern bool8 sub_803F428(DungeonPos *pos);
 extern void sub_800EEE0(u16 a0);
 extern s32 sub_800EBC8(struct UnkStruct_8040094 *a0);
 extern bool8 sub_800E7D0(struct UnkStruct_8040094 *a0);
-extern u8 sub_800EC84(s32 param_1);
+extern bool8 sub_800EC84(s32 moveId);
 extern bool8 MoveMatchesBideClassStatus(Entity *pokemon, Move *move);
 extern bool8 IsSleeping(Entity *pokemon);
 extern void sub_80421C0(Entity *pokemon, u16);
 
 extern const s32 gUnknown_8106A8C[];
 
-u16 sub_80412E0(u16 moveId, u8 weather, bool32 a2);
-static bool32 sub_804143C(Entity *entity, Move *move);
+u16 getEffectiveMoveId(u16 moveId, u8 weather, bool32 hasSpecialEffect);
```

Comment:
This a little detour from our naming  style. Was expecting GetEffectiveMoveId.

Reply:
My bad, fixed it